pytket/docs/circuit.rst

```diff
@@ -7,7 +7,7 @@ pytket.circuit
    circuit_class.rst
    optype.rst
    classical.rst
-   display.rst
+   display.md
```

Comment:
Are you planning to change the others to markdown as well?

Reply:
Not in a rush to do that.

I updated `display.md` because I wanted to use the MyST-NB library to show the code snippets on that page (consistent styling for the rest of the docs)

I updated the extensions index page because I wanted to use the more robust linking syntax where I don't have to use the full URL.

There are automated tools for doing the conversion so I could possibly update the other files in a subsequent PR.